modules/features2d/include/opencv2/features2d.hpp

```diff
@@ -724,8 +758,9 @@ class CV_EXPORTS_W SimpleBlobDetector : public Feature2D
   };
 
   CV_WRAP static Ptr<SimpleBlobDetector>
-    create(const SimpleBlobDetector::Params &parameters = SimpleBlobDetector::Params());
+    create(const SimpleBlobDetector::Params &parameters = SimpleBlobDetector::Params(), bool collectContours = false);
```

Comment:
shouldn't we include collectContours into SimpleBlobDetector::Params?

Reply:
i thought of `params` as a descriptor for which blobs should be detected, and `collectContours` as a modifier for the process itself. i think it's ok to include it into `params`, made changes.